src/pages/observability-packs.js

```diff
@@ -351,6 +352,24 @@ const ObservabilityPacksPage = ({ data, location }) => {
           `}
         `}
       >
+        <a
+          href="https://github.com/newrelic/newrelic-observability-packs"
+          target="_blank"
+          rel="noreferrer"
+          css={css`
+            text-decoration: none;
+          `}
+        >
+          <PackTile
+            css={css`
+              height: 100%;
```

Comment:
I think we'll need to do something to address the position in the list view (right now it's doing some weird things).

Reply:
This should be fixed now!